.gitignore

```diff
@@ -6,4 +6,4 @@ node_modules/
 build/
 
 # macOS
-.DS_Store5
\ No newline at end of file
+.DS_Store
```

Comment:
Hah! Nice catch. 👏

Reply:
I'll give `git status` the credit for that one haha